ui/src/components/Nodes/NodeDetailsDialog.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+  <FeatherDialog :modelValue="visible" relative :labels="labels" @update:modelValue="$emit('close')">
```

Comment:
I know this works, but I think it causes an `Extraneous non-emits event listeners` warning in the console unless you actually `defineEmits(['close'])` in the JS. Seems redundant but that's the only way I've found to get rid of them.

Reply:
Will try that!

Reply:
I didn't get that error in Chrome, but adding `defineEmits` in upcoming PR.